ci/posttest.yml

```diff
@@ -43,10 +43,10 @@ unix ui changes:
     - core device R test
     - legacy device test
   script:
-    - $NIX_SHELL --run "poetry run python -m tests.ui_tests.reporting master-diff"
-    - mv tests/ui_tests/reports/master_diff .
+    - $NIX_SHELL --run "poetry run python -m tests.ui_tests.reporting main-diff"
+    - mv tests/ui_tests/reports/main_diff .
```

Comment:
the command is still called `master-diff` and the path is hardcoded to `master_diff` it seems.
honestly i'd maybe keep those names? it seems more readable than "main_diff", which sounds more like "the main one of all the various diffs" than "diff from main branch".
(or rename it to "diff-from-main")

Reply:
O, I'll check it.

Reply:
f93d9a636d93370dfa68ecb49d1eb7dd8b236627